client/app/lib/providers/computestorage.coffee

```diff
@@ -173,6 +173,10 @@ module.exports = class ComputeStorage extends kd.Object
 
     postPush?.call this, value
 
+    @emit 'change', @storage
```

Comment:
let's not send the storage with the event instead we can send changed data like `"change:#{type}", { operation: 'push', value }` case but with addition `{type}`